src/wasm/syscalls_openmp.cpp

```diff
@@ -25,6 +25,19 @@ namespace wasm {
         return util::getUsableCores();
     }
 
+    WAVM_DEFINE_INTRINSIC_FUNCTION(env, "__kmpc_push_num_threads", void, __kmpc_push_num_threads, 
+                                   I32 loc, I32 global_tid, I32 num_threads) {
+        const std::shared_ptr<spdlog::logger> &logger = util::getLogger();
+        logger->debug("S - __kmpc_push_num_threads {} {}", global_tid, num_threads);
```

Comment:
Include all arguments passed to the function in these logging calls (should be 3 here and 1 in __kmpc_global_thread_num)